lib/google_search_data_viewer/keyword/google_search_parser.ex

```diff
@@ -0,0 +1,50 @@
+defmodule GoogleSearchDataViewer.Keyword.GoogleSearchParser do
+  @css_search_selectors %{
+    top_adwords: "#tads > .uEierd a.sVXRqc",
+    top_non_adwords: ".MhgNwc a",
+    non_adwords: ".yuRUbf a",
+    bottom_adwords: "#bottomads .uEierd a.sVXRqc"
+  }
+
+  def parse_html_urls(html) do
+    {_, parsed_html} = Floki.parse_document(html)
+
+    []
+    |> prepend_top_adwords(parsed_html)
+    |> prepend_top_non_adwords(parsed_html)
+    |> prepend_non_adwords(parsed_html)
+    |> prepend_bottom_adwords(parsed_html)
```

Comment:
I am not too sure about the word `prepend`. Technically, it is not wrong but it is focusing on an implementation of the Elixir language 😆 The following would be fine:

```suggestion
    |> parse_bottom_adwords(parsed_html)
```

Reply:
I have a habit of doing this, I do it in my commit messages too instead of talking in terms of project / domain. Ty